variables.tf

```diff
@@ -30,6 +30,14 @@ variable "ebs_endpoint" {
   description = "Variables to provision an EBS endpoint to the VPC"
 }
 
+variable "s3_endpoint" {
+  type = object({
+    route_table_ids = list(string)
+  })
```

Comment:
Why does this need to be an object?

Reply:
this would be passed into the module for additional subnets built outside of the module , we have a separate module also internally for create private and public subnets

Reply:
I get that... But why does it need to be an object?

Reply:
was following the existing pattern. will remove